samples/core/Miscellaneous/NewInEFCore9/QuerySample.cs

```diff
@@ -64,6 +73,20 @@ async Task<List<Post>> GetPostsForceConstant(int id)
                 .ToListAsync();
         #endregion
 
+        Console.WriteLine();
+        Console.WriteLine("Force constant primitive collection:");
+        Console.WriteLine();
+
+        _ = await GetPostsForceConstantCollection([1, 2, 3]);
+
+        #region ForceConstantPrimitiveCollection
+        async Task<List<Post>> GetPostsForceConstantCollection(int[] ids)
```

Comment:
maybe better do something like:

```cs
var ids = [1, 2, 3];
var query = context.Posts
    .Where(e => e.Title == ".NET Blog" && EF.Constant(ids).Contains(e.Id))
    .ToListAsync();
```

rather than via method call? @roji